cluster/gce/util.sh

```diff
@@ -3206,6 +3348,18 @@ function kube-down() {
         --quiet \
         "${MASTER_NAME}-ip"
     fi
+
+    if [[ ${GCE_PRIVATE_CLUSTER:-} == "true" ]]; then
+      delete-internal-loadbalancer
+    fi
+
+    if gcloud compute addresses describe "${MASTER_NAME}-internal-ip" --region "${REGION}" --project "${PROJECT}" &>/dev/null; then
```

Comment:
Same for this one, right?

Reply:
They both are in if [[ REMAINING_MASTER_COUNT -eq 0 ]] starting in 3340 and ending in 3363? :)